### PR TITLE
docs: Update dataset page

### LINKED ISSUE #3534: [DOCS] New page about making changes to a dataset
We are slowly adding workflows to make it possible to edit a dataset already present in Argilla.

This new page should cover things like:
- Adding more records
- Delete existing records
- Adding suggestions to existing records

### PR DESCRIPTION
**Pull request by nataliaElv**
# Description

New page to showcase different workflows to update and make changes to an existing Argilla dataset.

Closes #3534

**Type of change**

(Remember to title the PR according to the type of change)

- [x] Documentation update

**How Has This Been Tested**

(Please describe the tests that you ran to verify your changes.)

- [x] `sphinx-autobuild` (read [Developer Documentation](https://docs.argilla.io/en/latest/community/developer_docs.html#building-the-documentation) for more details)

**Checklist**

- [ ] I added relevant documentation
- [x] I followed the style guidelines of this project
- [x] I did a self-review of my code
- [x] I made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I filled out [the contributor form](https://tally.so/r/n9XrxK) (see text above)
- [x] I have added relevant notes to the `CHANGELOG.md` file (See https://keepachangelog.com/)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ These are the section headers that we use:
 - Added `pull` method in `RemoteFeedbackDataset` (a `FeedbackDataset` pushed to Argilla) to pull all the records from it and return it as a local copy as a `FeedbackDataset` ([#3465](https://github.com/argilla-io/argilla/pull/3465)).
 - Added `delete` method in `RemoteFeedbackDataset` (a `FeedbackDataset` pushed to Argilla) ([#3512](https://github.com/argilla-io/argilla/pull/3512)).
 - Added `delete_records` method in `RemoteFeedbackDataset`, and `delete` method in `RemoteFeedbackRecord` to delete records from Argilla ([#3526](https://github.com/argilla-io/argilla/pull/3526)).
+- Added new `Update a Feedback dataset` page to the documentation with information on how to make changes and update a `FeedbackDataset` ([#3535](https://github.com/argilla-io/argilla/pull/3535)).
```
Comment (alvarobartt) [inline CHANGELOG.md]: Should we add documentation changes in the `CHANGELOG.md`? cc @gabrielmbmb

File: docs/_source/guides/llms/practical_guides/update_dataset.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+:::
+
+:::{tab-item} Lower than Argilla 1.14.0
+```python
+# load the dataset
+dataset = rg.FeedbackDataset.from_argilla(name="my_dataset", workspace="my_workspace")
+# loop through the records and add suggestions
+for record in dataset.records:
+    record.set_suggestions([...])
+```
```
Comment (alvarobartt) [inline docs/_source/guides/llms/practical_guides/update_dataset.md]:
```suggestion
for record in dataset.records:
    record.set_suggestions([...])
dataset.push_to_argilla()
```

File: docs/_source/guides/llms/practical_guides/update_dataset.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+```python
+# load the dataset
+dataset = rg.FeedbackDataset.from_argilla(name="my_dataset", workspace="my_workspace")
+# loop through the records and update the suggestions
+for record in dataset.records:
+    record.update(suggestions=[...])
+```
+
+To learn about the schema that these suggestions should follow check [this page](create_dataset.md#add-suggestions).
```
Comment (alvarobartt) [inline docs/_source/guides/llms/practical_guides/update_dataset.md]: Indeed this section can be removed and mentioned as part of the `Add suggestions to existing records` section renaming it to `Add/Update suggestions in existing records`, as the code is exactly the same.

Comment (alvarobartt) [reply docs/_source/guides/llms/practical_guides/update_dataset.md]: If you prefer to keep it, then you should also add the `tab-set` with the different versions of the code-snippet

**Review by alvarobartt — COMMENTED** [review]
LGTM, just some minor changes/fixes 👍🏻

File: docs/_source/guides/llms/practical_guides/update_dataset.md (hunk 3 above)
Comment (nataliaElv) [reply docs/_source/guides/llms/practical_guides/update_dataset.md]: Yes, that's what I thought initially, but I wanted to do it simple like this first and check with you that was the case. I'll do that then 😃

**Pull request merged**